CONTRIBUTING.md

```diff
@@ -14,20 +14,15 @@ LTIAuthenticator requires Python >= 3.6.
    git clone https://github.com/jupyterhub/ltiauthenticator
    ```
 
-2. Do a development install with `pip`:
-
-   ```bash
-   cd ltiauthenticator
-   python3 -m pip install --editable .
-   ```
-
-3. Install the development requirements, which include things like linting and testing tools:
+2. Install the development requirements, which include things like linting and testing tools:
 
    ```bash
    python3 -m pip install -r dev-requirements.txt
```

Comment:
Now the instructions exclude actually installing the ltiauthenticator itself, thats a mistake right?

Reply:
Correct, the previous commit for this PR had a reference to install the package with `-e .` from a `requirements.txt` referenced in the `dev-requirements.txt` but removed that option since tests were red with that setup. The CONTRIBUTING guide has been updated to the previous version.